xml/deploy_cephadm.xml

```diff
@@ -1193,6 +1193,17 @@ config: OK
      deployment will use the intended cluster network from the start.
     </para>
    </note>
+   <para>
+     If your cluster is set to have dense nodes (greater than 60 OSDs per host),
```

Comment:
```suggestion
     If your cluster is set to have dense nodes (greater than 62 OSDs per host),
```
Just for consistency its probably better to use the same number? Apologies, should have thought of that in the initial proposal.